db/migrate/20150609193512_create_songs.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSongs < ActiveRecord::Migration
+  def change
+  	enable_extension 'uuid-ossp'
+
+    create_table :songs, id: :uuid do |t|
+      t.string :name
+      t.string :artist
+      t.string :spotify_uri
+      t.timestamps null: false
```

Comment:
The benefit to doing that is that when the actual database table is created, those `created_at` and `updated_at` are the first columns ... and they don't get lost somewhere in the middle as future migrations add on more columns.

So, it won't become

```
name
artist_id
spotify_uri
create_at
updated_at
likes_count
album_id
```

That way it's nice and consistent where they'll be.